pkg/service/rtcservice.go

```diff
@@ -236,27 +229,13 @@ func (s *RTCService) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// give it a few attempts to start session
 	var cr connectionResult
 	var initialResponse *livekit.SignalResponse
-	for attempt := 0; attempt < startConnectionMaxTries && r.Context().Err() == nil; attempt++ {
+	for attempt := 0; attempt < s.config.SignalRelay.ConnectAttempts; attempt++ {
 		connectionTimeout := 3 * time.Second * time.Duration(attempt+1)
```

Comment:
the existing backoff is bigger than the proposed exponential backoff and has the added benefit of doubling as a grace period for media response